axis_backend/apps/clients/models/activity.py

```
@@ -0,0 +1,107 @@
+        db_index=True,
+        help_text="Type of activity or interaction"
+    )
+    title = models.CharField(
+        max_length=255,
+        help_text="Brief activity title or summary"
+    )
+    description = models.TextField(
+        null=True,
+        blank=True,
+        help_text="Detailed activity description or notes"
+    )
```

Ruff (0.14.6)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)